pyccel/codegen/printing/fcode.py

```diff
@@ -285,7 +301,14 @@ def print_kind(self, expr):
         """
         Prints the kind(precision) of a literal value
```

Comment:
can you update the description of the method to include the updated info in it?

Reply:
sure!